.circleci/config.yml

```
@@ -31,9 +31,27 @@ jobs:
           command: |
             python ./src/sync_dashboards/main.py sync --config lookml_dashboards.yaml
           no_output_timeout: 15m
+  validate-lookml:
+    docker:
+      - image: circleci/node
```

This is a legacy image which is no longer being updated (the latest version has Node v17, which is several years old).  CircleCI [advises](https://circleci.com/docs/circleci-images/) using their next-generation `cimg/` images.

Also, I'd recommend pinning to avoid unexpected behavior changes.

```suggestion
      - image: cimg/node@22.7
```